src/System.Windows.Forms/src/System/Windows/Forms/NotifyIcon.cs

```diff
@@ -24,6 +24,7 @@ namespace System.Windows.Forms
     [SRDescription(nameof(SR.DescriptionNotifyIcon))]
     public sealed class NotifyIcon : Component
     {
+        internal const int MaxNotifyIconTextSize = 127;
```

Comment:
Since the const is in `NotifyIcon` having "NotifyIcon" again in the name doesn't add much value. 
```suggestion
        internal const int MaxTextLength = 127;
```

Reply:
Fixed